jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -528,6 +530,28 @@ public Builder setApplication(Application application) {
             _application = application;
             return this;
         }
+
+        /**
+         * Set the progress output file of the job expected to build.
+         *
+         * @param progressOutputFile {@link String} specifies the progress output file for the job.
+         * @return this builder.
+         */
+        public Builder setProgressOutputFile(String progressOutputFile) {
```

Comment:
How to use this one? Should it be relative path of MESOS_DIRECTORY ?

Reply:
I guess the logic is that the COOK executor will tailed the progress output file to and parse with regex to get the number?

Reply:
It can be an absolute path or a path relative to the sandbox directory. Cook executor will use the regex to match against lines from the progress output file (and stdout and stderr) to determine progress messages. I have updated the javadocs in the two methods.